fabric/src/main/java/io/github/gaming32/bingo/fabric/datagen/tag/BingoItemTagProvider.java

```diff
@@ -135,8 +136,9 @@ protected void addTags(HolderLookup.Provider registries) {
                 Items.WEEPING_VINES
             );
 
-        // The vanilla flower tag contains weird stuff like cherry leaves that bees are attracted to, but they are not flowers
-        getOrCreateTagBuilder(BingoItemTags.FLOWERS).forceAddTag(ItemTags.SMALL_FLOWERS).forceAddTag(ItemTags.TALL_FLOWERS);
+        getOrCreateTagBuilder(BingoItemTags.FLOWERS)
```

Comment:
Have you checked to see if there's any conventional item tags for these? I remember seeing some discussion about adding some on the Fabric discord.

Reply:
Yes, I did check.